docs/src/docs/dev-mode.md

```diff
@@ -0,0 +1,38 @@
+# Kit Dev: Run an LLM Locally
```

Comment:
Should we state somewhere that this is experimental feature? Perhaps even ask for feedback.